kubernetes/azure-machine-class.yaml

```diff
@@ -4,6 +4,7 @@ apiVersion: machine.sapcloud.io/v1alpha1
 kind: AzureMachineClass
 metadata:
   name: test-azure # Name of aws machine class goes here
+  namespace: test
```

Comment:
Same has to be added for AWS class example as well.

Reply:
Sure, make sense, missed it somehow.